### PR TITLE
Fixes for pycodestyle 2.2.0

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

File: contrib/nagios/check_galaxy.py [hunk 1]
```diff
@@ -347,7 +348,7 @@ def dprint(str):
     if debug:
         print(str)
 
-# do stuff here
```
Comment (natefoo) [inline contrib/nagios/check_galaxy.py:350]: ;D

File: test/unit/visualizations/plugins/test_VisualizationsRegistry.py [hunk 2]
```diff
@@ -89,11 +89,6 @@ def test_plugin_load( self ):
                     }
                 },
                 'not_a_vis1': {
-                    'config': {
-                        'vis1.xml': 'blerbler'
-                    },
-                },
-                'not_a_vis1': {
```
Comment (natefoo) [inline test/unit/visualizations/plugins/test_VisualizationsRegistry.py:96]: Should this be `not_a_vis5`?

Comment (nsoranzo) [reply test/unit/visualizations/plugins/test_VisualizationsRegistry.py:96]: Dunno, was like this from the very beginning of this file, the second dict entry happily shadowing the first one.

Comment (natefoo) [reply test/unit/visualizations/plugins/test_VisualizationsRegistry.py:96]: I suspect they were meant to be separate cases.

Comment (nsoranzo) [reply test/unit/visualizations/plugins/test_VisualizationsRegistry.py:96]: @natefoo Readded with `not_a_vis5` key.

**Pull request merged**